fastlane/Fastfile

```diff
@@ -25,7 +25,7 @@ SCREENSHOTS_DIR =  File.join(FASTLANE_DIR, 'screenshots')
 FASTLANE_METADATA_FOLDER = File.join(FASTLANE_DIR, 'metadata')
 WORKSPACE_PATH = File.join(PROJECT_ROOT_FOLDER, 'WooCommerce.xcworkspace')
 BUILDKITE_RELEASE_PIPELINE = 'release-builds.yml'
-IOS_LOCALES = %w[ar de-DE en-US es-ES fr-FR he id it ja ko nl-NL pt-BR ru sv tr zh-Hans zh-Hant].freeze
+IOS_LOCALES = %w[ar-SA de-DE en-US es-ES fr-FR he id it ja ko nl-NL pt-BR ru sv tr zh-Hans zh-Hant].freeze
```

Comment:
The promot screenshots generation except `ar-SA` and not just `ar`, and since everything else (metadata handling and also fastlane [deliver lane](https://docs.fastlane.tools/actions/deliver/)) expects `ar-SA`, I updated here for the screenshot generation step as well.